src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int quantity = 3;
```

Comment:
it's a final local variable, not a constant :) (should add static and move to the class scope)